tokens/themes/dark.json

```diff
@@ -609,6 +609,10 @@
             "value": "{palette.danger.800}",
             "type": "color"
           },
+          "foreground": {
+            "value": "{palette.danger.600}",
+            "type": "color"
+          },
```

Comment:
@gjones this property was missing from the dark mode file. I added the same property as in the light mode, so i guess we want to change it

Reply:
Sounds good, I'm pushing an updated version from figma now, it was a mislabled token.